chia/wallet/vault/vault_drivers.py

```diff
@@ -128,6 +128,13 @@ def match_vault_puzzle(mod: Program, curried_args: Program) -> bool:
     return False
 
 
+def match_p2_delegated_secp(mod: Program, curried_args: Program) -> bool:
```

Comment:
Is `curried_args` going to be used in future?

Reply:
I don't think so, but I think it's good to put them in here for a consistent API with other match functions.

Reply:
👍